_drafts/floating-point-101.md

```diff
@@ -0,0 +1,444 @@
+---
+title: Floating-point arithmetic – all you need to know, explained interactively
+image: /assets/floating-point-101/banner.png
+stylesheet: /assets/floating-point-101/number-line.css
+---
+
+Software engineering keeps getting more abstract, but one thing is unchanging: the importance of floating-point
+arithmetic. _Every_ computer programmer is bound to work with numbers (they call them _computers_ for a reason), so it's
+genuinely useful to understand the way machines do math, no matter if your code is for a to-do app, a stock exchange, or
+a fridge. How are numbers stored exactly? What's the significance of special values? And why is 0.1 + 0.2 not equal to
+0.3? Let's explore this all!
+
+## The Standard
```

Comment:
I felt the intro was going towards something exciting, but then immediately switched to a bit of history.

If it's true that we've settled on a standard, why bring this assumption to the surface at all?

Maybe it's worth being an appendix instead, if you want to go into alternatives to the standard?